include/pybind11/numpy.h

```diff
@@ -727,7 +728,9 @@ inline PYBIND11_NOINLINE void register_structured_dtype(
         if (field.offset > offset)
             oss << (field.offset - offset) << 'x';
         // note that '=' is required to cover the case of unaligned fields
```

Comment:
Minor, but maybe this comment line should be updated to reflect what's happening below